enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleWidget/MetabaseLinksToggleWidget.tsx

```diff
@@ -0,0 +1,30 @@
+import { t } from "ttag";
+
+import { Switch } from "metabase/ui";
+
+interface MetabaseLinksToggleSetting {
+  value: boolean | null;
+  default: boolean;
+}
+
+interface MetabaseLinksToggleWidgetProps {
+  setting: MetabaseLinksToggleSetting;
+  onChange: (value: boolean) => void;
+}
+
+export function MetabaseLinksToggleWidget({
+  setting,
+  onChange,
+}: MetabaseLinksToggleWidgetProps) {
+  return (
+    <>
+      <Switch
+        checked={setting.value ?? setting.default}
+        onChange={e => onChange(e.target.checked)}
+        labelPosition="left"
+        label={t`Show links and references`}
+        size="sm"
+      />
+    </>
```

Comment:
> 
```suggestion
    <Switch
      checked={setting.value ?? setting.default}
      onChange={e => onChange(e.target.checked)}
      labelPosition="left"
      label={t`Show links and references`}
      size="sm"
    />
```
Is the fragment a leftover or do we actually need it?

Reply:
Good catch. Not needed, it's probably left over before I turned it into a `Switch`